src/extensions/code/styles.css

```diff
@@ -1,9 +1,13 @@
 @import 'prismjs/themes/prism.css';
 
 /**
- * Add an additional `[class*=language-]` qualified selector to
+ * Add an additional `[class*=language-]` qualified selector for
  * `CodeFragment` and `CodeBlock` semantic selectors,
  * to match PrismJS’s CSS specificity, thereby allow styling of Prism tokenized elements.
  */
+
+/* prettier-ignore */
 @custom-selector :--CodeFragment [itemtype~='http://schema.stenci.la/CodeFragment'], [itemtype~='http://schema.stenci.la/CodeFragment'][class*=language-];
-@custom-selector :--CodeBlock [itemtype~='http://schema.stenci.la/CodeBlock'], [itemtype~='http://schema.stenci.la/CodeBlock'] > [class*=language-];
+
+/* prettier-ignore */
+@custom-selector :--CodeBlock [itemtype~='http://schema.stenci.la/CodeBlock'], [itemtype~='http://schema.stenci.la/CodeBlock'][class*=language-];
```

Comment:
I used `[itemtype~='http://schema.stenci.la/CodeBlock'] > [class*=language-]` because Encoda puts the class on the `<code>` element, not it's parent `<pre>` element. However, it seems that PrismJS decided to 'transfer' the class to the `pre`. I which case `[itemtype~='http://schema.stenci.la/CodeBlock'][class*=language-]` is fine and make more sense. Just noting that there is some magic there. All there more reason to move towards Web Components for `CodeBlocks` and `CodeFragments` I feel.